addons/test_discuss_full/tests/test_performance.py

```diff
@@ -488,6 +488,7 @@ def _get_init_messaging_result(self):
                 self._res_for_employee(self.users[14].employee_ids[0]),
             ],
             "Store": {
+                "has_unpinned_channels": False,
```

Comment:
I'd be surprised if test doesn't need +1 query with this change.
Unless I should have decreased it more in the previous PR?
Anyway the comments about queries should be adapted to add the new one.